scripts/get_all_heron_paths.sh

```diff
@@ -23,7 +23,8 @@ function query() {
 
 set +e
 # Build everything
-bazel build {heron,integration-test,tools/java}/...
+source ./detect_os_type.sh
```

Comment:
will this work when run from both the repo root dir (i.e. `scripts/get_all_heron_paths.sh`) and from within `scripts/.`?